Tools/Scripts/webkitpy/port/base.py

```diff
@@ -134,7 +140,10 @@ def __init__(self, host, port_name, options=None, **kwargs):
         self._options = options or optparse.Values()
 
         if self._name and '-wk2' in self._name:
-            self._options.webkit_test_runner = True
+            self._options.driver_name = self.DRIVER_NAMES[0]
```

Comment:
How come we aren't looking at `options.driver_names` here?

Reply:
We should be, this got update right after your feedback.